Unreviewed non-unified build fix

obyknovenius opened the pull request:
#### 89ad2f9c9b27cee4a5c06e91a40751974ecfee70
<pre>
Unreviewed non-unified build fix

* Source/WebCore/Modules/async-clipboard/ClipboardItemDataSource.h:
* Source/WebCore/Modules/indexeddb/server/MemoryIndex.cpp:
(WebCore::IDBServer::MemoryIndex::objectStore):
(WebCore::IDBServer::MemoryIndex::protectedObjectStore):
* Source/WebCore/Modules/indexeddb/server/MemoryIndex.h:
(WebCore::IDBServer::MemoryIndex::objectStore): Deleted.
(WebCore::IDBServer::MemoryIndex::protectedObjectStore): Deleted.
* Source/WebCore/css/CSSCursorImageValue.h:
* Source/WebCore/css/CSSValue.cpp:
* Source/WebCore/css/DOMMatrixReadOnly.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Align.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+CounterStyles.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+CounterStyles.h:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Grid.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Lists.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Lists.h:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+ViewTransition.cpp:
* Source/WebCore/dom/FragmentDirectiveUtilities.h:
* Source/WebCore/inspector/agents/InspectorDOMDebuggerAgent.cpp:
* Source/WebCore/platform/graphics/controls/EmptyControlFactory.cpp:
* Source/WebCore/platform/graphics/filters/FilterImage.h:
* Source/WebCore/rendering/LineClampUpdater.h:
* Source/WebCore/rendering/svg/SVGPaintServerHandling.h:
* Source/WebCore/rendering/svg/SVGTextBoxPainter.cpp:
* Source/WebCore/style/StyleAdjuster.h:
* Source/WebCore/workers/service/context/ServiceWorkerThreadProxy.cpp:
* Source/WebKit/NetworkProcess/Downloads/DownloadMonitor.h:
* Source/WebKit/UIProcess/MediaKeySystemPermissionRequestProxy.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebUserMediaClient.cpp:

Canonical link: <a href="https://commits.webkit.org/283555@main">https://commits.webkit.org/283555@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/6d3c97330e269deeebda37d06dc9ca122c953611

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/66411 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/45786 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/19032 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/70444 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/17543 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/68529 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/53585 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/17304 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/53267 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/11874 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/69478 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/42210 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/57493 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/33920 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/38881 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/14881 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/15897 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/60780 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/15223 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/72147 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/10368 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/14592 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/60591 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/10400 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/57562 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/60905 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/14703 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/8559 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/2174 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/41593 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/42670 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/43853 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/42413 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aperezdc reviewed (approved):
LGTM, although a r+ is not needed for unreviewed build fixes 😉

obyknovenius committed:
Unreviewed non-unified build fix

* Source/WebCore/Modules/async-clipboard/ClipboardItemDataSource.h:
* Source/WebCore/Modules/indexeddb/server/MemoryIndex.cpp:
(WebCore::IDBServer::MemoryIndex::objectStore):
(WebCore::IDBServer::MemoryIndex::protectedObjectStore):
* Source/WebCore/Modules/indexeddb/server/MemoryIndex.h:
(WebCore::IDBServer::MemoryIndex::objectStore): Deleted.
(WebCore::IDBServer::MemoryIndex::protectedObjectStore): Deleted.
* Source/WebCore/css/CSSCursorImageValue.h:
* Source/WebCore/css/CSSValue.cpp:
* Source/WebCore/css/DOMMatrixReadOnly.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Align.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+CounterStyles.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+CounterStyles.h:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Grid.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Lists.cpp:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Lists.h:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+ViewTransition.cpp:
* Source/WebCore/dom/FragmentDirectiveUtilities.h:
* Source/WebCore/inspector/agents/InspectorDOMDebuggerAgent.cpp:
* Source/WebCore/platform/graphics/controls/EmptyControlFactory.cpp:
* Source/WebCore/platform/graphics/filters/FilterImage.h:
* Source/WebCore/rendering/LineClampUpdater.h:
* Source/WebCore/rendering/svg/SVGPaintServerHandling.h:
* Source/WebCore/rendering/svg/SVGTextBoxPainter.cpp:
* Source/WebCore/style/StyleAdjuster.h:
* Source/WebCore/workers/service/context/ServiceWorkerThreadProxy.cpp:
* Source/WebKit/NetworkProcess/Downloads/DownloadMonitor.h:
* Source/WebKit/UIProcess/MediaKeySystemPermissionRequestProxy.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebUserMediaClient.cpp:

Canonical link: https://commits.webkit.org/283555@main
